client/my-sites/domains/domain-management/dns/dns-records.jsx

```diff
@@ -127,7 +127,7 @@ class DnsRecords extends Component {
 		if (
 			( ! englishLocales.includes( getLocaleSlug() ) &&
 				! i18n.hasTranslation(
-					'DNS records requires using WordPress.com nameservers. {{a}}Update your nameservers now{{/a}}.'
+					"Your domain is using external name servers so the DNS records you're editing won't be in effect until you switch to use WordPress.com name servers {{a}}Update your nameservers now{{/a}}."
```

Comment:
```suggestion
					"Your domain is using external name servers so DNS records updates won't be in effect until you switch to WordPress.com name servers. {{a}}Update your name servers now{{/a}}."
```
What do you think of this suggestion?